src/scripts/main.js

```diff
@@ -1,3 +1,55 @@
 'use strict';
 
-// write your code here
+function transformNumber(n) {
+  let bilions = Math.floor(n / 1000000000);
+  let milions = Math.floor((n - bilions * 1000000000) / 1000000);
+  let thousands
+    = Math.floor((n - bilions * 1000000000 - milions * 1000000) / 1000);
+  let hundreds
+    = n - bilions * 1000000000 - milions * 1000000 - thousands * 1000;
+
+  if (bilions >= 1) {
+    bilions += ',';
+  } else {
+    bilions = '';
+  }
+
+  if (milions >= 1) {
+    milions += ',';
+  } else {
+    milions = '';
+  }
```

Comment:
This is too complicated read about [Intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat)